website/src/pages/index.module.css

```diff
@@ -26,24 +26,32 @@
 
 .heroBanner {
   min-height: calc(100vh - 60px);
-  background-image: url('/img/banner-image.png');
+  background-color: #004643;
```

Comment:
You can add the color code as a css variable and use it here.

Reply:
Added color codes as css variables.